style.css

```diff
@@ -68,3 +75,337 @@ body {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  max-width: 327px;
+}
+
+.langs {
+  display: flex;
+  justify-content: left;
+  max-width: 327px;
+  margin-left: 0;
+}
+
+.langs div {
+  padding: 10px 12px;
+  max-width: 95px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  margin: 0 12px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.disp-btn {
+  display: flex;
+  justify-content: left;
+  width: 327px;
+  border: none;
+}
+
+.pst-btn {
+  background: #ff6b00;
+  margin: 12px 0;
+  border: none;
+  height: 48px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.projects {
+  align-self: flex-start;
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 8px;
+  font-family: Crete Round, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: 64px  0 0 12px;
+  padding-top: 12px;
+}
+
+.projects-details {
+  align-self: flex-start;
+  width: 295px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 12px;
+}
+
+.langs-2 {
+  display: flex;
+  justify-content: left;
+  width: 100%;
+  margin-left: 0;
+}
+
+.langs-2 div {
+  padding: 10px 12px;
+  max-width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  margin: 0 12px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.work-2 {
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.proj-work {
+  width: 327px;
+  background-image: url(/img/Mask-Group.png);
+  background-repeat: no-repeat;
+  margin: 24px;
+}
+
+.proj-btn {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  border: none;
+  height: 48px;
+  background: #ff6b00;
+  margin: 12px 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.see {
+  border: none;
+  background: #ff6b00;
+  margin: 12px 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about {
+  display: grid;
+  grid-template-rows: 652px 441.46px 360px 447.54px;
+  margin-top: 120px;
+  background-color: #1c1a19;
+}
```

Comment:
- [ ] I believe this is the class used for the about section. The content of this section should be aligned with flexbox.
- [ ] according to the original design this section has a background image. Your project is missing it kindly add it.